src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/packages.config

```diff
@@ -2,6 +2,9 @@
 <packages>
   <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Storage" version="5.0.0-preview" targetFramework="net45" />
```

Comment:
We are not able to use adapter implementation as it does not expose some properties which we need for our commandlet. We have seen similar patterns in other projects also.